staff/serena-mejias/e-commerce-template/style.css

```diff
@@ -0,0 +1,165 @@
+html {
+  box-sizing: border-box;
+  font-size: 14px;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+header {
+  position: fixed;
+  top: 0;
+  background-color: #dedede;
+  height: 120px;
+  width: 100%;
+  padding: 20px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  z-index: 99;
+}
+.icon{
+display: flex;
+flex-direction: row;
+color:white;
+flex: 1.5 ;
+}
+.icon-name{
```

Comment:
If I checked it wright the class icon-name is given to an h6 tag, so you would not need flexbox in it to display it properly. You could try to do it from its container (icon class)

Reply:
Done!